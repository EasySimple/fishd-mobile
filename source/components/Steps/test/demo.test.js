import webDemoTest from '../../../tests/demoTest';

webDemoTest('Steps');
