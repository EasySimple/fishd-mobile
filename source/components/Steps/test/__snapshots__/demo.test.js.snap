// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Steps demo correctly 1`] = `
<div>
  <div
    class="components-tpl-demo-basic"
  >
    <div
      class="demo-title"
    >
      基础用法
    </div>
    <div
      class="demo-card"
    >
      <div
        class="fm-steps fm-steps__horizontal"
      >
        <div
          class="fm-steps__items"
        >
          <div
            class="fm-steps__step fm-steps__step-process"
          >
            <div
              class="fm-steps__title"
              style="color:#07c160"
            >
              <div>
                买家下单
              </div>
            </div>
            <div
              class="fm-steps__dot"
            >
              <i
                class="icon iconfont fm-icon fm-icon-fm-wancheng"
                style="width:14px;height:14px;color:#07c160"
              />
            </div>
            <div
              class="fm-steps__line"
            />
          </div>
          <div
            class="fm-steps__step"
          >
            <div
              class="fm-steps__title"
              style="color:#969799"
            >
              <div>
                商家接单
              </div>
            </div>
            <div
              class="fm-steps__dot"
            >
              <i
                class="fm-steps__icon"
                style="background-color:#969799"
              />
            </div>
            <div
              class="fm-steps__line"
            />
          </div>
          <div
            class="fm-steps__step"
          >
            <div
              class="fm-steps__title"
              style="color:#969799"
            >
              <div>
                买家提货
              </div>
            </div>
            <div
              class="fm-steps__dot"
            >
              <i
                class="fm-steps__icon"
                style="background-color:#969799"
              />
            </div>
            <div
              class="fm-steps__line"
            />
          </div>
          <div
            class="fm-steps__step"
          >
            <div
              class="fm-steps__title"
              style="color:#969799"
            >
              <div>
                交易完成
              </div>
            </div>
            <div
              class="fm-steps__dot"
            >
              <i
                class="fm-steps__icon"
                style="background-color:#969799"
              />
            </div>
            <div
              class="fm-steps__line"
            />
          </div>
        </div>
      </div>
      <a
        aria-disabled="false"
        class="fm-button fm-button-default fm-button-small"
        role="button"
      >
        <span
          class="fm-button-text"
        >
          下一步
        </span>
      </a>
    </div>
  </div>
  <div
    class="components-tpl-demo-basic"
  >
    <div
      class="demo-title"
    >
      自定义颜色
    </div>
    <div
      class="demo-card"
    >
      <div
        class="fm-steps fm-steps__horizontal"
      >
        <div
          class="fm-steps__items"
        >
          <div
            class="fm-steps__step fm-steps__step-process"
          >
            <div
              class="fm-steps__title"
              style="color:blue"
            >
              <div>
                买家下单
              </div>
            </div>
            <div
              class="fm-steps__dot"
            >
              <i
                class="icon iconfont fm-icon fm-icon-fm-wancheng"
                style="width:14px;height:14px;color:blue"
              />
            </div>
            <div
              class="fm-steps__line"
            />
          </div>
          <div
            class="fm-steps__step"
          >
            <div
              class="fm-steps__title"
              style="color:#969799"
            >
              <div>
                商家接单
              </div>
            </div>
            <div
              class="fm-steps__dot"
            >
              <i
                class="fm-steps__icon"
                style="background-color:#969799"
              />
            </div>
            <div
              class="fm-steps__line"
            />
          </div>
          <div
            class="fm-steps__step"
          >
            <div
              class="fm-steps__title"
              style="color:#969799"
            >
              <div>
                买家提货
              </div>
            </div>
            <div
              class="fm-steps__dot"
            >
              <i
                class="fm-steps__icon"
                style="background-color:#969799"
              />
            </div>
            <div
              class="fm-steps__line"
            />
          </div>
          <div
            class="fm-steps__step"
          >
            <div
              class="fm-steps__title"
              style="color:#969799"
            >
              <div>
                交易完成
              </div>
            </div>
            <div
              class="fm-steps__dot"
            >
              <i
                class="fm-steps__icon"
                style="background-color:#969799"
              />
            </div>
            <div
              class="fm-steps__line"
            />
          </div>
        </div>
      </div>
      <a
        aria-disabled="false"
        class="fm-button fm-button-default fm-button-small"
        role="button"
      >
        <span
          class="fm-button-text"
        >
          下一步
        </span>
      </a>
    </div>
  </div>
  <div
    class="components-tpl-demo-basic"
  >
    <div
      class="demo-title"
    >
      自定义图标
    </div>
    <div
      class="demo-card"
    >
      <div
        class="fm-steps fm-steps__horizontal"
      >
        <div
          class="fm-steps__items"
        >
          <div
            class="fm-steps__step fm-steps__step-process"
          >
            <div
              class="fm-steps__title"
              style="color:blue"
            >
              <div>
                买家下单
              </div>
            </div>
            <div
              class="fm-steps__dot"
            >
              <i
                class="icon iconfont fm-icon fm-icon-fm-star"
                style="width:14px;height:14px;color:blue"
              />
            </div>
            <div
              class="fm-steps__line"
            />
          </div>
          <div
            class="fm-steps__step"
          >
            <div
              class="fm-steps__title"
              style="color:#969799"
            >
              <div>
                商家接单
              </div>
            </div>
            <div
              class="fm-steps__dot"
            >
              <i
                class="icon iconfont fm-icon fm-icon-fm-moreinfo-copy"
                style="width:14px;height:14px;color:#ccc"
              />
            </div>
            <div
              class="fm-steps__line"
            />
          </div>
          <div
            class="fm-steps__step"
          >
            <div
              class="fm-steps__title"
              style="color:#969799"
            >
              <div>
                买家提货
              </div>
            </div>
            <div
              class="fm-steps__dot"
            >
              <i
                class="icon iconfont fm-icon fm-icon-fm-moreinfo-copy"
                style="width:14px;height:14px;color:#ccc"
              />
            </div>
            <div
              class="fm-steps__line"
            />
          </div>
          <div
            class="fm-steps__step"
          >
            <div
              class="fm-steps__title"
              style="color:#969799"
            >
              <div>
                交易完成
              </div>
            </div>
            <div
              class="fm-steps__dot"
            >
              <i
                class="icon iconfont fm-icon fm-icon-fm-moreinfo-copy"
                style="width:14px;height:14px;color:#ccc"
              />
            </div>
            <div
              class="fm-steps__line"
            />
          </div>
        </div>
      </div>
      <a
        aria-disabled="false"
        class="fm-button fm-button-default fm-button-small"
        role="button"
      >
        <span
          class="fm-button-text"
        >
          下一步
        </span>
      </a>
    </div>
  </div>
  <div
    class="components-tpl-demo-basic"
  >
    <div
      class="demo-title"
    >
      竖向步骤条
    </div>
    <div
      class="demo-card"
    >
      <div
        class="fm-steps fm-steps__vertical"
      >
        <div
          class="fm-steps__items"
        >
          <div
            class="fm-steps__step fm-steps__step-process"
          >
            <div
              class="fm-steps__title"
              style="color:#07c160"
            >
              <div>
                买家下单
              </div>
              <div
                class="fm-steps__description"
              >
                2020-09-01 12:40
              </div>
            </div>
            <div
              class="fm-steps__dot"
            >
              <i
                class="icon iconfont fm-icon fm-icon-fm-wancheng"
                style="width:14px;height:14px;color:#07c160"
              />
            </div>
            <div
              class="fm-steps__line"
            />
          </div>
          <div
            class="fm-steps__step"
          >
            <div
              class="fm-steps__title"
              style="color:#969799"
            >
              <div>
                商家接单
              </div>
              <div
                class="fm-steps__description"
              >
                2020-09-01 13:40
              </div>
            </div>
            <div
              class="fm-steps__dot"
            >
              <i
                class="fm-steps__icon"
                style="background-color:#969799"
              />
            </div>
            <div
              class="fm-steps__line"
            />
          </div>
          <div
            class="fm-steps__step"
          >
            <div
              class="fm-steps__title"
              style="color:#969799"
            >
              <div>
                买家提货
              </div>
              <div
                class="fm-steps__description"
              >
                2020-09-01 15:20
              </div>
            </div>
            <div
              class="fm-steps__dot"
            >
              <i
                class="fm-steps__icon"
                style="background-color:#969799"
              />
            </div>
            <div
              class="fm-steps__line"
            />
          </div>
          <div
            class="fm-steps__step"
          >
            <div
              class="fm-steps__title"
              style="color:#969799"
            >
              <div>
                交易完成
              </div>
              <div
                class="fm-steps__description"
              >
                2020-09-01 16:50
              </div>
            </div>
            <div
              class="fm-steps__dot"
            >
              <i
                class="fm-steps__icon"
                style="background-color:#969799"
              />
            </div>
            <div
              class="fm-steps__line"
            />
          </div>
        </div>
      </div>
      <a
        aria-disabled="false"
        class="fm-button fm-button-default fm-button-small"
        role="button"
      >
        <span
          class="fm-button-text"
        >
          下一步
        </span>
      </a>
    </div>
  </div>
</div>
`;
